### DISCUSSION TITLE
subdivide for volumes

### DISCUSSION THREAD

**Discussion by j042** [tataratat/gustaf#169] [General]
hexa would be 1 elem -> 8 elem,
tet would go from 1 elem -> 4 elem. ([here's reference](https://en.wikipedia.org/wiki/Tetrahedron_in_Bottrop))

**Comment by j042** [comment]
@jzwar @clemens-fricke @danielwolff1

**Comment by jzwar** [comment]
Not sure what your intention is here? Subdivision for simplices?
